lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -427,6 +431,25 @@ newtype ApiMaintenanceAction = ApiMaintenanceAction
     { gcStakePools :: ApiT PoolMetadataGCStatus
     } deriving (Eq, Generic, Show)
 
+data ApiAsset = ApiAsset
+    { policyId :: ApiT W.TokenPolicyId
+    , policyItem :: ApiT W.TokenName
```

Comment:
If we're using the names in the specification, perhaps should this be "assetName"?

(Apologies if I'm missing some context here.)

```suggestion
    , assetName :: ApiT W.TokenName
```